code/modules/ninja/outfit.dm

```diff
@@ -1,6 +1,6 @@
 /datum/outfit/ninja
 	name = "Space Ninja"
-	uniform = /obj/item/clothing/under/color/black
+	uniform = /obj/item/clothing/under/syndicate
```

Comment:
This change brings a collateral melee resistance buff to space ninjas, make a no sensors black jumpsuit subtype, please.